src/modules/partition/core/PartitionLayout.cpp

```diff
@@ -239,6 +241,26 @@ PartitionLayout::execute( Device* dev,
         {
             currentPartition->fileSystem().setLabel( part.partLabel );
         }
+        if ( !part.partFeatures.isEmpty() )
+        {
+            for ( const auto &k : part.partFeatures.keys() )
+            {
+                const auto &v = part.partFeatures.value(k);
```

Comment:
Nitpick: `const auto& v` is the official-Calamares-style for that. (You can use ci/calamaresstyle to apply astyle+clang-format with the right bits set)

Reply:
Oops, my bad. Fixed!